index.ts

```diff
@@ -42,7 +44,7 @@ const userSchema = z.object({
     .optional(),
 });
 
-type User = z.infer<typeof userSchema>;
+export interface User extends z.infer<typeof userSchema> {}
```

Comment:
Just so that typescript doesnt show the entire type definition on hover and instead just show `User`